x-pack/plugins/cloud_security_posture/server/lib/fleet_util.ts

```diff
@@ -41,6 +44,9 @@ const isPolicyTemplate = (input: any): input is CloudSecurityPolicyTemplate =>
 const getPackageNameQuery = (): string => {
   return `${PACKAGE_POLICY_SAVED_OBJECT_TYPE}.package.name:"${CLOUD_SECURITY_POSTURE_PACKAGE_NAME}"`;
 };
+const getPaginatedItems = <T>(filteredItems: T[], page: number, perPage: number) => {
```

Comment:
nit: would be nice to cover `getPaginatedItems` and `filterPackagesByCriteria` with unit tests